src/app/app.component.scss

```diff
@@ -1 +1,4 @@
 @import "styles/main";
+body{
+  background-color: #000d36;
```

Comment:
Create a variable in the scss structure or use an exisiting color variable..

Reply:
@lunayach Please address this

Reply:
Actually, it got it here by mistake. Removed it.